### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by Arbina001**
### Headings typography
Screenshot 2023-12-19 at 11 04 02 PM

### Font Colors
Screenshot 2023-12-19 at 11 04 58 PM

### CODE REVIEW

File: src/stories/Typography/typography.stories.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { TextContainer } from './Typography.style';
+
+export default {
+  title: 'Typography',
+  component: Typography,
+};
+
+export const HeadingTypography = () => (
+  <TextContainer>
+    <Typography headingTag="h1" text="H1. Adorn Box heading" />
```
Comment (tarakhant1297) [inline src/stories/Typography/typography.stories.js:12]: @Arbina001 give proper name to `headingTag ` follow antd Typography props name

File: src/stories/Typography/typography.style.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+const baseStyles = css`
+  color: ${({ type }) => theme?.[type]};
+  font-weight: bold;
+  font-family: sans-serif;
+`;
+
+const size = {
+  h1: {
+    fontSize: '3rem',
```
Comment (tarakhant1297) [inline src/stories/Typography/typography.style.js:12]: @Arbina001 read from theme

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/stories/Typography/typography.stories.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import Typography from './Typography';
+import { TextContainer} from './Typography.style';
+
+export default {
+  title: 'Typography',
+}; 
+
+export const HeadingTypography = () => (
+  <TextContainer>
+    <Typography htmlTag="h1" text="H1. Adorn Box heading" />
```
Comment (tarakhant1297) [inline src/stories/Typography/typography.stories.js:11]: <Typography htmlTag="h1" >H1. Adorn Box heading</Typography>

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/stories/Typography/typography.style.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    lineHeight: '2.05rem',
```
Comment (ankitadrixus08) [inline src/stories/Typography/typography.style.js:13]: line-height generally same as font or greater than this @Arbina001

**Review by ankitadrixus08 — APPROVED** [review]

**Review by ankitadrixus08 — APPROVED** [review]

**Review by tarakhant1297 — APPROVED** [review]

**Pull request merged**